templates/zerver/invalid_realm.html

```diff
@@ -0,0 +1,9 @@
+{% extends "zerver/portico.html" %}
+{% block portico_content %}
+
+<h3>{{ _('Realm Does Not Exist') }}</h3>
+
+<p>{{ _('Hi there! Thank you for your interest in Zulip.') }}.</p>
+<p>{% trans %}The organization you are trying to access does not have a Zulip setup at this address.{% endtrans %}.</p>
```

Comment:
We're using "organization".  I actually changed that message already on zulipchat.com; I think it's probably not worth anyone else working on this PR until I have a chance to clean up our fork of it.